EpsilonLights/Src/Lights.c

```diff
@@ -154,6 +151,49 @@ void blinkSignalLightsTask(void const* arg)
     }
 }
 
+void updateStrobeLight(void const* arg)
+{
+    uint32_t prevWakeTime = osKernelSysTick();
+    // Store inputs values
+    // NOTE: All Lights Out pins are active low
+
+    // If blinkerTimer is within (0 - BLINKER_FREQ), turn blinkers on
+    // If blinkerTimer is within (BLINKER_FREQ - BLINKER_FREQ*2), keep blinkers off
+    // If blinkerTimer is greater than (BLINKER_FREQ*2) reset blinkerTimer to 0
+    uint32_t blinkerTimer = 0;
+    char strobeLight;
+
+    for (;;)
+    {
+        osDelayUntil(&prevWakeTime, LIGHTS_UPDATE_FREQ);
+        //  headlightsOff = (lightsInputs >> HOFF_INPUT_INDEX) & 1;
+        //  brakes = (driversInputs[BRAKES_INPUT_INDEX_P1] >> BRAKES_INPUT_INDEX_P2) & 1;
+        strobeLight = (auxBmsInputs[1] >> 0) & 1;
```

Comment:
is 1 supposed to be the strobe_fault_mask?

Reply:
Yea

Reply:
Should i use a variable instead?

Reply:
Yes please!